docs/OV_Runtime_UG/Samples_Overview.md

```diff
@@ -142,8 +142,8 @@ cd build
 make
 ```
 
-For the release configuration, the sample application binaries are in `<path_to_build_directory>/intel64/Release/`;
-for the debug configuration — in `<path_to_build_directory>/intel64/Debug/`.
+For the release configuration, the sample application binaries are in `<path_to_build_directory>/<hardware_name>/Release/`;
+for the debug configuration — in `<path_to_build_directory>/<hardware_name>/Debug/`.
```

Comment:
=> architecture 

in other places as well

Reply:
we're on it ;)